.drone.yml

```diff
@@ -82,6 +82,9 @@ pipeline:
       GS_CLIENT_EMAIL: ${GS_CLIENT_EMAIL}
       GS_PRIVATE_KEY: ${GS_PRIVATE_KEY}
       DOMAIN: ${CI_DOMAIN}
+      SYSLOG_SERVER: ${SYSLOG_SERVER}
```

Comment:
You can use `service containers`  with drone. http://readme.drone.io/0.5/usage/services/ 

In this case you could deploy a syslog service container for running tests.